### PR TITLE
[docs] [CLOUDGA-11169] Add Automation section and CLI

### PR DESCRIPTION
**Pull request by ddhodge**
Add YugabyteDB Managed CLI reference and Automation section

@netlify /preview/yugabyte-cloud/managed-automation/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+
+You can pass these values as flags. For example:
+
+```sh
+ybm --apikey AWERDFSSS --host cloud.yugabyte.com cluster get
+```
+
+For convenience, configure `ybm-cli` with default values for these flags as follows:
+
+- Using a configuration file called `.ybm-cli.yaml` under your `$HOME` directory. You can use the command `ybm configure` to set up the file. For example:
```
Comment (shinji62) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md:60]: Currently, `.ybm-cli.yaml` is the default file name, but user can specify a different file with the flag `--config` for example.
```
ybm --config ~/.ybm-cli-portal.yaml cluster get
```

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md [hunk 2]
```diff
@@ -0,0 +1,697 @@
...
+
+- `YBM_APIKEY`
+
+  The API key to use to authenticate to your YugabyteDB Managed account.
+
+- `YBM_HOST`
+
+  The YugabyteDB Managed host.
+
+## Examples
```
Comment (posriniv) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md:693]: This is good for now. However, I test all the commands, provide more examples and update this doc if needed.

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md [hunk 3]
```diff
@@ -0,0 +1,697 @@
...
+Using a shell script:
+
+```sh
+curl -sSL https://downloads.yugabyte.com/get_ybm_cli.sh | bash
+```
+
+If you have wget, you can use the following:
+
+```sh
+wget -q -O - https://downloads.yugabyte.com/get_ybm_cli.sh | sh
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md:46]: I stuck these here as placeholders.

File: docs/content/preview/yugabyte-cloud/managed-automation/_index.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    parent: yugabytedb-managed
+    identifier: managed-automation
+    weight: 600
+type: indexpage
+---
+
+Use the following automation tools to manage your YugabyteDB Managed account and clusters:
+
+| Automation | Description |
+| :--- | :--- |
```
Comment (polarweasel) [inline docs/content/preview/yugabyte-cloud/managed-automation/_index.md:19]: Trivial change to make Jira notice the issue link :)
```suggestion
| :--------- | :---------- |
```

**Review by cdavid — APPROVED** [review]

File: docs/content/preview/yugabyte-cloud/managed-automation/_index.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+---
+title: Automation
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/_index.md:2]: Make title unique.

File: docs/content/preview/yugabyte-cloud/managed-automation/_index.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+### Authentication
+
+For access, automation tools require authentication in the form of an [API key](managed-apikeys/).
+
+### Account details
+
+For some REST API commands, you may need one or more of the following account details:
+
+- Your YugabyteDB Managed account ID. To view your account ID, click the **Profile** icon in the top right corner of the YugabyteDB Managed window. The account ID is also displayed in the API Key Details.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/_index.md:32]:
```suggestion
- Your YugabyteDB Managed account ID. To view your account ID, click the **Profile** icon in the top right corner of the YugabyteDB Managed window. The account ID is also displayed in the API key Details.
```

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-apikeys.md [hunk 7]
```diff
@@ -1,14 +1,16 @@
 ---
 title: API keys
 headerTitle:
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-apikeys.md:3]: headerTitle empty ?

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/_index.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+  </div>
+
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="managed-cli-reference">
+      <div class="head">
+        <img class="icon" src="/images/section_icons/deploy/enterprise/console.png" aria-hidden="true" />
+        <div class="title">Reference</div>
+      </div>
+      <div class="body">
+        ybm CLI command reference.
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/_index.md:41]: I think CLI implies command.

**Review by aishwarya24 — COMMENTED** [review]
some comments

File: docs/content/preview/yugabyte-cloud/managed-automation/_index.md (hunk 5 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/managed-automation/_index.md:2]: there's another page with this title?

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-read-replica.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+| --cluster-name | Required. Name of the cluster whose read replicas you want to fetch. |
+
+### update
+
+Update a read replica for a specified cluster.
+
+| Flag | Description |
+| :--- | :--- |
+| --cluster-name | Required. Name of the cluster with read replicas you want to update. |
+| --replica | Specifications for the read replica provided as key-value pairs.<br>Arguments:<br><ul><li>num_cores - number of vCPUs per node. Default is 2.</li><li>memory_mb - memory (MB) per node. Default is 4096.</li><li>disk_size_gb - disk size (GB) per node. Default is 10.</li><li>code - cloud provider (`AWS` or `GCP`)</li><li>region - region in which to deploy the read replica</li><li>num_nodes - number of nodes for the read replica. Default is 1.</li><li>vpc_name - name of the VPC in which to deploy the read replica</li><li>num_replicas - the replication factor</li><li>multi-zone - whether the read replica is multi-zone (`true` or `false`). Default is false.</li></ul>
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-read-replica.md:77]:
```suggestion
| --replica | Specifications for the read replica provided as key-value pairs.<br>Arguments:<br><ul><li>num_cores - number of vCPUs per node. Default is 2.</li><li>memory_mb - memory (MB) per node. Default is 4096.</li><li>disk_size_gb - disk size (GB) per node. Default is 10.</li><li>code - cloud provider (`AWS` or `GCP`)</li><li>region - region in which you want to deploy the read replica</li><li>num_nodes - number of nodes for the read replica. Default is 1.</li><li>vpc_name - name of the VPC in which you want to deploy the read replica</li><li>num_replicas - the replication factor</li><li>multi-zone - whether the read replica is multi-zone (`true` or `false`). Default is false.</li></ul>
```

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-read-replica.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+## Commands
+
+### create
+
+Create a read replica for a specified cluster.
+
+| Flag | Description |
+| :--- | :--- |
+| --cluster-name | Required. Name of the cluster to which you want to add read replicas. |
+| --replica | Specifications for the read replica provided as key-value pairs.<br>Arguments:<br><ul><li>num_cores - number of vCPUs per node. Default is 2.</li><li>memory_mb - memory (MB) per node. Default is 4096.</li><li>disk_size_gb - disk size (GB) per node. Default is 10.</li><li>code - cloud provider (`AWS` or `GCP`)</li><li>region - region in which to deploy the read replica</li><li>num_nodes - number of nodes for the read replica. Default is 1.</li><li>vpc_name - name of the VPC in which to deploy the read replica</li><li>num_replicas - the replication factor</li><li>multi-zone - whether the read replica is multi-zone (`true` or `false`). Default is false.</li></ul>
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-read-replica.md:52]: This terse formulation is fine; keeping the word count down in tight corners like this

**Pull request merged**